presto/testing/integration_tests/analyze_tables.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import prestodb
+
+
+def analyze_tables(presto_cursor, schema_name, verbose=False):
+    """Analyze all tables in the given schema to collect statistics.
+    
+    Args:
+        presto_cursor: Presto database cursor
+        schema_name: Name of the schema containing tables to analyze
+        verbose: If True, print detailed progress information
+    """
+    try:
+        if verbose:
+            print(f"Discovering tables in schema '{schema_name}'...")
+        tables = presto_cursor.execute(f"SHOW TABLES FROM hive.{schema_name}").fetchall()
+        table_names = [table_name for table_name, in tables]
+        
+        if not table_names:
+            print(f"Warning: No tables found in schema '{schema_name}'")
+            return
+        
+        if verbose:
+            print(f"Found {len(table_names)} table(s): {', '.join(table_names)}")
+            print(f"\nStarting table analysis...")
+        
+        success_count = 0
+        failure_count = 0
+        
+        for i, table_name in enumerate(table_names, 1):
+            try:
+                if verbose:
+                    print(f"  [{i}/{len(table_names)}] Analyzing table '{table_name}'...", end=" ")
+                presto_cursor.execute(f"ANALYZE hive.{schema_name}.{table_name}")
+                if verbose:
+                    print("OK")
+                success_count += 1
+            except Exception as e:
+                if verbose:
+                    print("FAILED")
+                print(f"Warning: Failed to analyze table '{table_name}': {e}")
+                failure_count += 1
+        
+        if verbose:
```

Comment:
Nit: Does `if verbose or failure_count > 0:` also work here instead of the `elif`?